puppet/modules/web/templates/rsync_downloads.sh.erb

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -e
+
+case "$SSH_ORIGINAL_COMMAND" in
+*\&*)
+echo "Rejected"
```

Comment:
Not sure I looked at this earlier, but could you indent the `echo` lines?

Reply:
dude, I just copied that from some other file, but sure :)

Reply:
Are you suggesting that not all files are in perfect condition?

Reply:
I'd never suggest such a blasphemy!